torch/_dynamo/polyfills/_collections.py

```diff
@@ -0,0 +1,29 @@
+"""
+Python polyfills for builtins
+"""
+
+from collections.abc import Iterable, MutableMapping
+from typing import Any
+
+from ..decorators import substitute_in_graph
+
+
+__all__ = []
+
+try:
+    import _collections  # type: ignore[import-not-found]
+
+    @substitute_in_graph(_collections._count_elements)
+    def _count_elements(
+        mapping: MutableMapping[Any, int],
+        iterable: Iterable[Any],
```

Comment:
This should match the TypeVar of the previous argument right? Use an explicit TypeVar to enforce that.